### ISSUE TITLE
KeyError: 'blackbox'

### ISSUE THREAD

**Issue by ruizlenato** [Stunkymonkey/nautilus-open-any-terminal#58]
OS: Arch Linux
Gnome verson: 42.4
Nautilus version: 42.2
the blackbox was installed using the aur repositories 

Log from terminal:
```
➜  ~ nautilus                                                                          
Traceback (most recent call last):
  File "/home/renato/.local/share/nautilus-python/extensions/open_any_terminal_extension.py", line 255, in <module>
    new_tab = bool(NEW_TAB_PARAMS[value] is not None)
KeyError: 'blackbox'
Initializing Nextcloud-client-nautilus extension

```

**Comment by Stunkymonkey** [comment]
how do you start blackbox via command line? what version of this plugin is installed?

**Comment by kimlulz** [comment]
Same error to me..
```
BlackBox version 0.12.0
GNOME Shell 42.4
GNOME nautilus 42.2
```

**Comment by Stunkymonkey** [comment]
are you using the flatpack variant?

**Comment by Stunkymonkey** [comment]
and what version of this package are you using?

**Comment by Stunkymonkey** [comment]
I guess this is fixed by using flatpack in #60

**Comment by kimlulz** [comment]
Sry
I was using blackbox from aur(arch user repo) that was flagged..
Change to flakpak and it works!
